.github/workflows/repo2cpe-download.yaml

```diff
@@ -0,0 +1,37 @@
+name: Download Repo2CPE
+on:
+  push:
+    branches:
+    - "yli3/**"
+  schedule:
+  - cron: "0 */3 * * *"
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Authenticate with Google Cloud
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+
+    - name: Create Repo2CPE directory
+      run: mkdir -p Repo2CPE
+
+    - name: Download repository-to-cpe.json
+      run: |
+        cd Repo2CPE
+        wget -q https://access.redhat.com/security/data/metrics/repository-to-cpe.json
+
+    - name: Check downloaded file
+      run: ls -l Repo2CPE/repository-to-cpe.json
+
+    - name: Upload file to Google Cloud Storage
+      run: gsutil cp -r "Repo2CPE" "gs://scanner-v4-test/"
```

Comment:
What happens if uploads are interrupted? What if the Repo2CPE files are bad? 
Would it make sense to upload a versioned list, e.g. with upload datetime as suffix in the directory name and create a link to the latest ?

Reply:
I think the best way is to inform scanner on-call person if the upload fails at this stage. And I will probably need your help to set up a Slack token for the workflow @tommartensen 

cc: @jvdm @RTann

Reply:
You can use it like https://github.com/stackrox/stackrox/blob/master/.github/workflows/cut-rc.yml#L276-L300 in your action as well. 
`SLACK_BOT_TOKEN` is a GHA secret, `channel-id` is the short ID of the channel, e.g. `C033Z8KMZAM` for `forum-acs-scanner`.
 
If you mention `@acs-scanner-primary` in the text, they will get a notification.

Reply:
@gavin-stackrox  was suggesting that curl command is good enough to post messages on Slack. Also I go with secrets.SLACK_ONCALL_SCANNER_WEBHOOK , which is ACS CI Notification in scanner-oncall channel.

Reply:
Fine for me. Just link one of the failed test runs and a successful one as "proof" in the PR description.